RecoBTag/ImpactParameter/python/pfNegativeOnlyJetProbabilityBJetTags_cfi.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+pfNegativeOnlyJetProbabilityBJetTags = cms.EDProducer("JetTagProducer",
```

Comment:
For the JetTagProducers cloning would not be that useful since all parameters are typically updated. So these I won't modify.